lib/Backends/NNPI/tests/NNPIDeviceManagerTest.cpp

```diff
@@ -18,7 +18,20 @@
 using namespace glow;
 
 std::set<std::string> glow::backendTestBlacklist = {
-    "MultiFunction/0",
-    "DeviceResidentTensors/0",
-    "TransferStaticPlaceholderTest/0",
-};
+    "MultiFunction/0", "DeviceResidentTensors/0",
+    "TransferStaticPlaceholderTest/0", "CanHandleDeviceResidentTensors/0"};
```

Comment:
nit: Please add a comma after the last item here so they are clang-formatted to one per line.